rcqt.so/src/rpmPackageInfo.cpp

```diff
@@ -35,11 +35,17 @@ bool RpmPackageInfo::readMetaPackageInfo(std::string ss){
   bool found = false;
   size_t pos = std::string::npos;
   std::ofstream os;
+  size_t relPos = std::string::npos;
 
   os.open(getFileName() , std::ofstream::out | std::ofstream::app);
 
   while(std::getline(inss, line)){
 
+    /* Exclude rpmlib* packages that are not part ROCm */
+    if(0 == line.find("rpmlib")) {
```

Comment:
is it guaranteed that rpmlib packages start with this pattern always? Or is comparing against npos better?

Reply:
Yes, thats what I have seen. Those packages start with rpmlib*.